generator/build.hxml

```diff
@@ -10,6 +10,7 @@
 -lib version
 -lib tink_template
 -lib utest
+-lib hxnodejs
```

Comment:
Ops, this breaks the generated client `.js` files, as they try to find a suitable node env and fail.